cmd/gateway-gcs.go

```diff
@@ -631,6 +633,9 @@ func (l *gcsGateway) ListMultipartUploads(bucket string, prefix string, keyMarke
 }
 
 func fromGCSMultipartKey(s string) (key, uploadID string, partID int, err error) {
+	// remove prefixes
+	s = filepath.Base(s)
```

Comment:
filepath.Base doesn't seem to work on all operating systems. I saw its internal code and it depends on os.IsPathSeparator which is '/' on unix and '\' on Windows.